From 9145b51c1cc4e9ecd273e8eb41de1bbcdd607e35 Mon Sep 17 00:00:00 2001
From: "Friedrich W. H. Kossebau" <kossebau@kde.org>
Date: Thu, 28 Jun 2012 20:22:41 +0200
Subject: [PATCH] Remove image/x-wmf and image/x-xfig from image thumbnailer's
 supported mimetypes

There is no known QImageIOPlugin for these two mimetypes.
And the image thumbnailer currently might hide a special thumbnailer for these,
as the code of the thumbnail creation invoking code just takes the first
in the list of found thumbnail plugins, cmp. ThumbnailProtocol::pluginForMimeType(...)
in kde-runtime/kioslave/thumbnail/thumbnail.cpp

REVIEW: 105371
---
 kioslave/thumbnail/imagethumbnail.desktop |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kioslave/thumbnail/imagethumbnail.desktop b/kioslave/thumbnail/imagethumbnail.desktop
index 17263a9..b4ea0f4 100644
--- a/kioslave/thumbnail/imagethumbnail.desktop
+++ b/kioslave/thumbnail/imagethumbnail.desktop
@@ -68,6 +68,6 @@ Name[x-test]=xxImages (GIF, PNG, BMP, ...)xx
 Name[zh_CN]=图像 (GIF、PNG、BMP ...)
 Name[zh_TW]=影像（GIF、PNG、BMP...）
 X-KDE-ServiceTypes=ThumbCreator
-MimeType=image/cgm;image/fax-g3;image/gif;image/jp2;image/png;image/tiff;image/bmp;image/x-dds;image/x-ico;image/x-jng;image/x-pcx;image/x-photo-cd;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-rgb;image/x-tga;image/x-wmf;image/x-xbitmap;image/x-xcf;image/x-xfig;image/x-xpixmap;image/x-sun-raster;image/vnd.adobe.photoshop;image/x-psd;image/x-hdr;image/x-pic;image/vnd.microsoft.icon;image/x-icon;
+MimeType=image/cgm;image/fax-g3;image/gif;image/jp2;image/png;image/tiff;image/bmp;image/x-dds;image/x-ico;image/x-jng;image/x-pcx;image/x-photo-cd;image/x-portable-bitmap;image/x-portable-graymap;image/x-portable-pixmap;image/x-rgb;image/x-tga;image/x-xbitmap;image/x-xcf;image/x-xpixmap;image/x-sun-raster;image/vnd.adobe.photoshop;image/x-psd;image/x-hdr;image/x-pic;image/vnd.microsoft.icon;image/x-icon;
 X-KDE-Library=imagethumbnail
 CacheThumbnail=true
-- 
1.7.10.4

